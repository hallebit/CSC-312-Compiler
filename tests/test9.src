let remainder =
    fun y ->
        fix f x ->
            if (y <= x - y) then
                f (x - y)
            else 
                x - y
    in (remainder 3) 10
