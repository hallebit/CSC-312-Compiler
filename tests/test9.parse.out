(let remainder:int -> int -> int = (fun (y:int) : int -> int -> (fix f (x:int) : int -> (if (y <= (x - y)) then (f (x - y)) else (x - y)))) in ((remainder 3) 10))